components/brave_welcome_ui/components/text/index.ts

```diff
@@ -47,3 +47,14 @@ export const Paragraph = styled.p`
     }
   }
 `
+
+export const TermsOfService = styled(Paragraph)`
+  font-size: 13px;
+  line-height: 20px;
+  opacity: .8;
+
+  /* Collapse bottom margin of preceding Paragraph */
+  ${Paragraph} + & {
+    margin-top: -19px;
```

Comment:
The intent here is to reduce the whitespace between the main panel `Paragraph` (which has a bottom margin of 32px) and the terms-of-service links immediately below. The negative margin is unfortunate, but minimizes the amount of change to code within the welcome UI.

Another option would be to change `Paragraph` into a more generic container (`<div>` instead of a `<p>`) and allow the terms-of-service to sit within it. A downside of that approach is that we can't independently set the opacity for terms-of-service text.